src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -323,9 +335,20 @@ private void filter(final VariantContext vc, final FeatureContext featureContext
         writer.add(builder.make());
     }
 
-    private GenotypesContext makeGenotypes(final VariantContext vc) {
+    private GenotypesContext makeGenotypes(final VariantContext vc, final VariantContextBuilder builder ) {
         final GenotypesContext genotypes = GenotypesContext.create(vc.getGenotypes().size());
 
+        //
+        // recompute AC, AN and AF if filtered genotypes are set to no-call
+        //
+        // occurrences of alternate alleles over all genotypes
+        final Map<Allele, Integer> calledAltAlleles = new LinkedHashMap<Allele, Integer>(vc.getNAlleles()-1);
```

Comment:
why is the capacity set to `vc.getNAlleles()-1` if you know you're putting `vc.getAlternateAlleles().size()` keys into the map?